pulsarlib/api.go

```diff
@@ -433,3 +446,39 @@ func ListPartionedTopics(tenantID string, namespace string) ([]string, error) {
 	}
 	return result, nil
 }
+
+func CreateSubscriptionOnTopic(accountUUID, clusterName, indexName, position string) error {
```

Comment:
This is a library function. It doesn't care if we use for elasticsearch or something else
It should take following arguments
tenantId, Namespace, topicName, subscriptionName, subscriptionType, position